src/App.vue

```diff
@@ -1,8 +1,11 @@
 <script setup>
 import HelloWorld from './components/HelloWorld.vue'
+import PokeDex from './components/PokeDex.vue'
 </script>
 
 <template>
-  <HelloWorld msg="Hello Světe!"/>
+  <HelloWorld msg="Pokédex in progress!"/>
+
+  <PokeDex></PokeDex>
```

Comment:
If you mean <PokeDex></PokeDex> then I don't understand how can I shorten it. Because it is atypical to do i real life, I am not able to find any clue on the internet. Or you do you mean to delete the empty row?

Reply:
Sorry, suddenly I understand. No need for answer.

Reply:
<PokeDex />

Reply:
< pokedex />